From 8bcf77e9cc0eeb48c753924b08873d7649a02a8f Mon Sep 17 00:00:00 2001
From: Jon Mulder <jon.e.mulder@gmail.com>
Date: Tue, 19 Mar 2024 20:15:32 -0400
Subject: [PATCH] Rename to min_queued_buffers

---
 fthd_v4l2.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/fthd_v4l2.c b/fthd_v4l2.c
index ffd8593..6c00ee4 100644
--- facetimehd-main/fthd_v4l2.c
+++ facetimehd-main/fthd_v4l2.c
@@ -683,7 +683,7 @@ int fthd_v4l2_register(struct fthd_private *dev_priv)
 	q->mem_ops = &vb2_dma_sg_memops;
 	q->buf_struct_size = 0;//sizeof(struct vpif_cap_buffer);
 	q->timestamp_flags = V4L2_BUF_FLAG_TIMESTAMP_MONOTONIC;
-	q->min_buffers_needed = 1;
+	q->min_queued_buffers = 1;
 	q->lock = &dev_priv->vb2_queue_lock;
 
 	ret = vb2_queue_init(q);
-- 
2.44.0

